community/resources/index.md

```diff
@@ -21,6 +21,13 @@ Does it fill gaps in our current docs and provide important information?
 **Note:** Magento does not officially endorse any of the blogs, books, presentation, or podcasts linked below.
 They are listed here because the content has been well received within the Magento community.
 </div>
```

Comment:
Can you add a blank line here? Between the </div> and new heading? It'll just append the text when generated without putting it into that format.

Reply:
I have added the line